### PR TITLE
[DX-1278] Gateway release notes 5.3.1

### PR DESCRIPTION
**Pull request by andyo-tyk**
### Preview Link
https://deploy-preview-4399--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-gateway/release-notes/version-5.3/

___

## **Type**
documentation

___

## **Description**
- Added a new section for Tyk Gateway 5.3.1 release notes, including comprehensive details such as release date, breaking changes, deprecations, and upgrade instructions.
- Updated the document tags to reflect the inclusion of version 5.3.1.
- Detailed changelog for version 5.3.1, categorizing changes into 'Changed' and 'Fixed' sections.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Addition of 5.3.1 Release Notes to Tyk Gateway Documentation</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md
<li>Added 5.3.1 release notes section with details on release date, <br>breaking changes, deprecations, upgrade instructions, release <br>highlights, downloads, and changelog.<br> <li> Updated tags to include "5.3.1".<br> <li> Provided detailed changelog for version 5.3.1, including changed and <br>fixed items.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4399/files#diff-ead598197a92e18014f4f7c35c341621948fb6cdf66d45c1dd05de21bc2b35aa">+63/-2</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

___

## **Type**
Documentation

___

## **Description**
- Added a comprehensive new section for Tyk Gateway version 5.3.1 in the release notes documentation.
- Included details such as release date, upgrade instructions, and a detailed changelog with specific bug fixes and enhancements.
- Updated document tags to reflect the new version 5.3.1.
- Provided download links and instructions for various platforms, enhancing accessibility for users.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Addition of Tyk Gateway 5.3.1 Release Notes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md
<li>Added a new section for Tyk Gateway 5.3.1 release notes.<br> <li> Updated document tags to include "5.3.1".<br> <li> Detailed changelog for version 5.3.1, categorizing changes into <br>'Fixed' and providing links to resources.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4399/files#diff-ead598197a92e18014f4f7c35c341621948fb6cdf66d45c1dd05de21bc2b35aa">+115/-2</a>&nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 1]
```diff
@@ -21,6 +21,93 @@ Our minor releases are supported until our next minor comes out.
...
+<summary>Policy-API link deletion code updated to ensure DocumentDB compatibility</summary>
+
+Policy-API link deletion code was updated to address a compatibility issue with DocumentDB. The previous version relied on the $expr operator, supported by MongoDB but not DocumentDB.
+</details>
+</li>
+<li>
+<details>
+<summary>Gateway panics addressed due to JSVM usage with ignore plugin</summary>
+
+Fixed a panic scenario occurring when JSVM is utilised alongside require_session:true and ignore auth middleware. While the JSVM middleware expects a valid session, the configuration flag doesn't guarantee its presence, only that it's passed if available.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
Fixed a panic scenario occurring when JSVM is utilised alongside require_session:true and ignore auth middleware. While the JSVM middleware expects a valid session, the `require_session` configuration flag doesn't guarantee its presence, only that it is passed if available.
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 2]
```diff
@@ -21,6 +21,93 @@ Our minor releases are supported until our next minor comes out.
...
+<li>
+<details>
+<summary>Tyk cache confusion leads to mixed GraphQL body responses</summary>
+
+GraphQL APIs were returning incorrect responses when simultaneous GQL calls with different inputs were made, which looked as if caching was mixed up. This was related to a setting in the GraphQL engine, that has now been turned off, so simultaneous GQL calls won't return an incorrect response.
+</details>
+</li>
+<li>
+<details>
+<summary>Gateway panics when arguments are missing in persist GraphQL endpoint operations</summary>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Gateway panics when arguments are missing in persist GraphQL endpoints</summary>
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
@andyo-tyk @Eopayemi I have made minor suggestions and uploaded some changes on  the branch to remove Sentence Casing from the change-log summary headings

Requires TBC fields for release date, download links and dependencies

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 3]
```diff
@@ -21,6 +21,93 @@ Our minor releases are supported until our next minor comes out.
...
+- [source code](TBC)
+
+### Changelog {#Changelog-v5.3.1}
+
+#### Fixed
+
+<ul>
+<li>
+<details>
+<summary>Security enhancement: rejecting unsigned plugin bundles in Gateway</summary>
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Security enhancement: don't load APIs into Gateway if custom plugin bundle fails to load</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 4]
```diff
@@ -21,6 +21,93 @@ Our minor releases are supported until our next minor comes out.
...
+
+Issues were addressed where Tyk failed to properly reject custom plugin bundles with signature verification failures, allowing APIs to load without necessary plugins, potentially exposing upstream services. With the fix, if plugin bundle loading fails (e.g., due to signature verification), the API will not be loaded and an error will be logged in the Gateway.
+</details>
+</li>
+<li>
+<details>
+<summary>Resolved quota limit issue with API URL Rewrite to Self and Policy-Created Key</summary>
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>URL Rewrite middleware did not always correctly observe quotas for requests using keys created from policies</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 5]
```diff
@@ -21,6 +21,93 @@ Our minor releases are supported until our next minor comes out.
...
+
+
+Fixed two bugs related to quota limits not being applied and incorrect reporting of remaining quota value in the X-RateLimitRemaining header when the API had URL rewrite middleware configured with a tyk://self loop.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
Fixed two bugs in the handling of usage quotas by the URL rewrite middleware when it was configured to rewrite to itself (e.g. to `tyk://self`). Quota limits were not observed and the quota related response headers always contained `0`. 
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 6]
```diff
@@ -21,6 +21,93 @@ Our minor releases are supported until our next minor comes out.
...
+</details>
+</li>
+<li>
+<details>
+<summary>Policy-API link deletion code updated to ensure DocumentDB compatibility</summary>
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
```

This is a Dashboard change - shouldn't be in this release note

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 7]
```diff
@@ -21,6 +21,93 @@ Our minor releases are supported until our next minor comes out.
...
+
+Policy-API link deletion code was updated to address a compatibility issue with DocumentDB. The previous version relied on the $expr operator, supported by MongoDB but not DocumentDB.
+</details>
+</li>
+<li>
+<details>
+<summary>Gateway panics addressed due to JSVM usage with ignore plugin</summary>
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Fixed a Gateway panic that could occur when using custom JavaScript plugins with the Ignore Authentication middleware</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
Fixed a panic scenario that occurred when a custom JavaScript plugin that requests access to the session metadata (`require_session:true`) is assigned to the same endpoint as the Ignore Authentication middleware. While the custom plugin expects access to a valid session, the configuration flag doesn't guarantee its presence, only that it's passed if available. As such, the custom plugin should be coded to verify that the session metadata is present before attempting to use it.
```

**Review by andyo-tyk — COMMENTED** [review]
Added some suggested improvements to Core APIM changes. Spotted one that should be in Dashboard RN and flagged for removal.

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 8]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+<li>
+<details>
+<summary>Gateway panics when arguments are missing in persist GraphQL endpoints</summary>
+
+In some instances users were noticing gateway panics when using "Persist GQL operation middleware" without arguments defined. This issue has been fixed and the gateway will not throw panics in these cases anymore.
+</details>
+</li>
+<li>
+<details>
+<summary>Incorrect MDCB Gateway Counter on License Statistics Page</summary>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Incorrect MDCB Gateway Counter on License Statistics page</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 9]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+
+Resolved an issue where the MDCB gateway counter inaccurately incremented when a gateway was stopped and restarted.
+</details>
+</li>
+<li>
+<details>
+<summary>Missing GraphQL OTel Attributes in Spans on Request Validation Failure</summary>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Missing GraphQL OTel attributes in Spans on Request Validation failure</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 10]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+
+In cases where detailed_tracing was set to false and client was sending a malformed request to a GQL API the traces were missing GQL attributes (operation name, type and document). This has been corrected and debugging GQL with OTel will be easier for users.
+</details>
+</li>
+<li>
+<details>
+<summary>API Cache Clearing from Dashboard Not Propagated to Edge Deployments</summary>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>API cache clearing from Dashboard not propagated to edge deployments</summary>
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks for updating change-log items @Eopayemi. I have made minor suggestions to update sentence case of change-log items.
Before approval dependencies, downloads and release date needs adding before release

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 11]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+### Changelog {#Changelog-v5.3.1}
+
+#### Fixed
+
+<ul>
+<li>
+<details>
+<summary>Security enhancement: don't load APIs into Gateway if custom plugin bundle fails to load</summary>
+
+Issues were addressed where Tyk failed to properly reject custom plugin bundles with signature verification failures, allowing APIs to load without necessary plugins, potentially exposing upstream services. With the fix, if plugin bundle loading fails (e.g., due to signature verification), the API will not be loaded and an error will be logged in the Gateway.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
Issues were addressed where Tyk failed to properly reject custom plugin bundles with signature verification failures, allowing APIs to load without necessary plugins, potentially exposing upstream services. With the fix, if the plugin bundle fails to load (for example, due to failed signature verification) the API will not be loaded and an error will be logged in the Gateway.
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 12]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+<summary>Fixed a Gateway panic that could occur when using custom JavaScript plugins with the Ignore Authentication middleware</summary>
+
+Fixed a panic scenario that occurred when a custom JavaScript plugin that requests access to the session metadata (`require_session:true`) is assigned to the same endpoint as the Ignore Authentication middleware. While the custom plugin expects access to a valid session, the configuration flag doesn't guarantee its presence, only that it's passed if available. As such, the custom plugin should be coded to verify that the session metadata is present before attempting to use it.
+</details>
+</li>
+<li>
+<details>
+<summary>Tyk cache confusion leads to mixed GraphQL body responses</summary>
+
+GraphQL APIs were returning incorrect responses when simultaneous GQL calls with different inputs were made, which looked as if caching was mixed up. This was related to a setting in the GraphQL engine, that has now been turned off, so simultaneous GQL calls won't return incorrect response
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
GraphQL APIs were returning incorrect responses when simultaneous GraphQL calls with different inputs were made, which looked as if caching was mixed up. This was related to a setting in the GraphQL engine, that has now been turned off, so simultaneous calls won't return an incorrect response.
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 13]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+</details>
+</li>
+<li>
+<details>
+<summary>Gateway panics when arguments are missing in persist GraphQL endpoints</summary>
+
+In some instances users were noticing gateway panics when using "Persist GQL operation middleware" without arguments defined. This issue has been fixed and the gateway will not throw panics in these cases anymore.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
In some instances users were noticing gateway panics when using the **Persist GQL** middleware without arguments defined. This issue has been fixed and the gateway will not throw panics in these cases anymore.
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md (hunk 8 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Tyk Dashboard License Statistics page could display incorrect number of data plane gateways</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 14]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+</details>
+</li>
+<li>
+<details>
+<summary>Incorrect MDCB Gateway Counter on License Statistics Page</summary>
+
+Resolved an issue where the MDCB gateway counter inaccurately incremented when a gateway was stopped and restarted.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
Resolved an issue in distributed deployments where the MDCB data plane gateway counter was inaccurately incremented when a Gateway was stopped and restarted.
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md (hunk 9 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Missing GraphQL OTel attributes in spans when requests fail validation</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 15]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+</details>
+</li>
+<li>
+<details>
+<summary>Missing GraphQL OTel Attributes in Spans on Request Validation Failure</summary>
+
+In cases where detailed_tracing was set to false and client was sending a malformed request to a GQL API the traces were missing GQL attributes (operation name, type and document). This has been corrected and debugging GQL with OTel will be easier for users.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
In cases where `detailed_tracing` was set to `false` and the client was sending a malformed request to a GraphQL API, the traces were missing GraphQL attributes (operation name, type and document). This has been corrected and debugging GraphQL with OTel will be easier for users.
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md (hunk 10 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Unable to clear the API cache in distributed data plane Gateways from the control plane Dashboard</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 16]
```diff
@@ -21,6 +21,114 @@ Our minor releases are supported until our next minor comes out.
...
+</details>
+</li>
+<li>
+<details>
+<summary>API Cache Clearing from Dashboard Not Propagated to Edge Deployments</summary>
+
+Addressed a bug where clearing the API cache from the Tyk Dashboard UI failed to invalidate the cache in distributed data plane gateways.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
Addressed a bug where clearing the API cache from the Tyk Dashboard failed to invalidate the cache in distributed data plane gateways.
```

**Review by andyo-tyk — COMMENTED** [review]
Proposed wording improvements

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 17]
```diff
@@ -21,6 +21,119 @@ Our minor releases are supported until our next minor comes out.
...
+<li>
+<details>
+<summary>Fixed a Gateway panic that could occur when using custom JavaScript plugins with the Ignore Authentication middleware</summary>
+
+Fixed a panic scenario that occurred when a custom JavaScript plugin that requests access to the session metadata (`require_session:true`) is assigned to the same endpoint as the Ignore Authentication middleware. While the custom plugin expects access to a valid session, the configuration flag doesn't guarantee its presence, only that it's passed if available. As such, the custom plugin should be coded to verify that the session metadata is present before attempting to use it.
+</details>
+</li>
+<li>
+<details>
+<summary>Resolved Gateway crashes with Python Plugin storage APIs</summary>
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
<summary>Gateway could crash when custom Python plugins attempted to access storage</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 18]
```diff
@@ -21,6 +21,119 @@ Our minor releases are supported until our next minor comes out.
...
+
+Fixed a bug causing Python plugin storage APIs store_data and get_data to fail due to connection issues with configured storage.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
Fixed a bug where the Gateway could crash when using custom Python plugins that access the Redis storage. The Tyk Python API methods `store_data` and `get_data` could fail due to connection issues with the Redis. With this fix, the Redis connection will be created if required, avoiding the crash.
```

**Review by andyo-tyk — COMMENTED** [review]
Tweak to wording to improve comprehension

**Review by dcs3spp — APPROVED** [review]
@andyo-tyk PR LGTM, I have added dependencies section with statement that links to 5.3.0 dependencies

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 19]
```diff
@@ -21,6 +21,123 @@ Our minor releases are supported until our next minor comes out.
...
+### Breaking Changes
+**Attention**: Please read this section carefully.
+
+There are no breaking changes in this release, however if moving from an version of Tyk older than 5.3.0 please read the explanation provided with [5.3.0 release]({{< ref "#TykOAS-v5.3.0">}}).
+
+### Deprecations
+There are no deprecations in this release.
+
+### Upgrade Instructions
+If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should skip 5.3.0 and upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: If user is upgrading from older version and skip 5.3.0, they need to review and follow 5.3.0 upgrade instructions too.

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 20]
```diff
@@ -21,6 +21,123 @@ Our minor releases are supported until our next minor comes out.
...
+
+Please refer to [dependencies](#dependencies-5.3.0) provided for release 5.3.0
+
+### Downloads
+- [Docker image to pull](https://hub.docker.com/r/tykio/tyk-gateway/tags?page=&page_size=&ordering=&name=v5.3.1)
+  - ```bash
+    docker pull tykio/tyk-gateway:v5.3.1
+    ``` 
+- Helm charts
+  - [tyk-charts GH Repo](https://github.com/TykTechnologies/tyk-charts/releases)
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: Latest helm chart release notes is in Docs: /product-stack/tyk-charts/release-notes/version-1.3

**Review by carolwkc — CHANGES_REQUESTED** [review]

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 21]
```diff
@@ -21,6 +21,123 @@ Our minor releases are supported until our next minor comes out.
...
+<summary>Tyk Dashboard License Statistics page could display incorrect number of data plane gateways</summary>
+
+Resolved an issue in distributed deployments where the MDCB data plane gateway counter was inaccurately incremented when a Gateway was stopped and restarted.
+</details>
+</li>
+<li>
+<details>
+<summary>Unable to clear the API cache in distributed data plane Gateways from the control plane Dashboard</summary>
+
+Addressed a bug where clearing the API cache from the Tyk Dashboard failed to invalidate the cache in distributed data plane gateways.
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: @andyo-tyk This one requires MDCB 2.5.1 is it? Should we note the dependency here?

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md (hunk 19 above)
Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: @caroltyk I have updated, if this and other comments are ok could you approve for release?

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md (hunk 21 above)
Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: @caroltyk I have updated PR to mention MDCB 2.5.1. If ok and other comments ok would be to approve for release?

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md (hunk 20 above)
Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: @caroltyk I have updated PR to mention MDCB 2.5.1. If ok and other comments ok would be to approve for release?

**Review by carolwkc — APPROVED** [review]
LGTM

**Pull request merged**